src/test/java/duke/DeadlineTest.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeadlineTest {
+
+    @Test
+    void toString_withIsoDate_formatsAsMonthName() {
```

Comment:
Consider using camel casing for method names?
```suggestion
    void toString_withIsoDate_formatsAsMonthName() {
```